environment/docker/php/Dockerfile

```diff
@@ -11,4 +11,4 @@ RUN curl -sS https://getcomposer.org/installer | php -- --version=${COMPOSER_VER
 RUN docker-php-ext-install mysqli pdo pdo_mysql json
```

Comment:
I would suggest to move docker-ext-configure right here and move intl to other extesions install part